### PR TITLE
feat(lock): add seven lock component

### PR DESCRIPTION
**Pull request by yefengxing**
## Owner

- [x] Nickname(Tuya): 风眠

## Description

add seven lock component

## Package

- [x] tuya-panel-lock-sdk

## Type of change

Please select the relevant option.

- [x] New feature (non-breaking change which adds functionality)

## Review CheckList

Don't edit this section.

- [ ] Intra-group review
- [x] ESLint lint
- [x] Example Run
- [x] Jest Run
- [x] No Grammer Error
- [x] No Spell Error

### CODE REVIEW

File: packages/tuya-panel-lock-sdk/src/count-down-button/hooks.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React, { useState, useRef } from 'react';
+import { CountDownTimerType, Timer } from './interface';
+
+export const useCountDownTimer: CountDownTimerType = (count = 3) => {
+  const [number, setNumber] = useState<number>(count);
+  const intervalRef = useRef<ReturnType<typeof setInterval>>();
+
+  const startCount = () => {
+    const coutDownFn = () => {
```
Comment (ZengBeauty) [inline packages/tuya-panel-lock-sdk/src/count-down-button/hooks.ts:9]: coutDownFn 拼写错误

File: packages/tuya-panel-lock-sdk/src/share/utils.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+const message = require('../res/message.png');
+const more = require('../res/more.png');
+const weChat = require('../res/weChat.png');
+export const initShareList = ['Message', 'Email', 'WeChat', 'Copy', 'More'];
+
+export const getShareList: any = () => {
+  const shareDataList = {
+    message: {
+      img: message,
+      title: '短信',
```
Comment (ZengBeauty) [inline packages/tuya-panel-lock-sdk/src/share/utils.ts:12]: 组件里面要用多语言

File: packages/tuya-panel-lock-sdk/src/share/shareManager.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+const ShareManager: React.FC<ShareMessageProps> = props => {
+  const {
+    themeColor,
+    shareData,
+    modalStyle,
+    onClick,
+    onFail,
+    onSuccess,
+    confirmText = '确定',
```
Comment (ZengBeauty) [inline packages/tuya-panel-lock-sdk/src/share/shareManager.tsx:19]: 多语言

File: packages/tuya-panel-lock-sdk/src/share/index.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    style,
+    textStyle,
+    themeColor = '#239C8E',
+    modalStyle,
+    onClick,
+    onFail,
+    onSuccess,
+    opacity,
+    children,
+    cancelText = '取消',
```
Comment (ZengBeauty) [inline packages/tuya-panel-lock-sdk/src/share/index.tsx:20]: 多语言

File: packages/tuya-panel-lock-sdk/src/time-picker-range/index.tsx [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+    defaultBeginTime = 480,
+    defaultEndTime = 1200,
+    beginTime: _beginTime,
+    endTime: _endTime,
+    endTimeLabel,
+    beginTimeLabel,
+    hourLabel = '',
+    minutesLabel = '',
+    secondLabel = '',
+    placeholder = '选择时间',
```
Comment (ZengBeauty) [inline packages/tuya-panel-lock-sdk/src/time-picker-range/index.tsx:31]: 多语言

File: packages/tuya-panel-lock-sdk/src/custom-keyboard/interface.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { StyleProp, TextStyle, ViewStyle } from 'react-native';
```
[the remark this answers is not in the document]
Comment (youngjuning) [reply packages/tuya-panel-lock-sdk/src/custom-keyboard/interface.ts]: 无用的引用尽量删除

File: packages/tuya-panel-lock-sdk/src/time-picker-range/index.tsx (hunk 5 above)
Comment (yefengxing) [reply packages/tuya-panel-lock-sdk/src/time-picker-range/index.tsx]: 已修复

File: packages/tuya-panel-lock-sdk/src/count-down-button/hooks.ts (hunk 1 above)
Comment (yefengxing) [reply packages/tuya-panel-lock-sdk/src/count-down-button/hooks.ts]: 已修复

File: packages/tuya-panel-lock-sdk/src/custom-keyboard/interface.ts (hunk 6 above)
[the remark this answers is not in the document]
Comment (yefengxing) [reply packages/tuya-panel-lock-sdk/src/custom-keyboard/interface.ts]: 已删除

**Pull request merged**